frontend/src/routes/MonthlyWorkingHours.jsx

```diff
@@ -10,6 +10,7 @@ const MonthlyWorkingHours = () => {
   const [selectedYearsBack, setSelectedYearsBack] = useState(3);
   const [graphData, setGraphData] = useState([]);
 
+  const { Title } = Typography;
```

Comment:
Usually these lines are moved outside of the component, because they never change. So this could be moved to line 7. And then const MonthlyWorkingHours should start on line 9.